content/en/getting-started/api-key.md

```diff
@@ -0,0 +1,85 @@
+
+{{< alert title="API Key Security" >}}
+- Avoid sharing your API key with anyone. Ensure that you do not commit it to any source code management systems (like Git repositories).
+
+- If you push an API key to a public repository, it has potentially been exposed and might remain in the history (even if you try to rewrite it).
+
+- If you accidentally publish your API key, [contact us](https://localstack.cloud/contact/) to get your API key rotated!
+
+- If you want to use your API key in your CI environment, check out our [CI documentation]({{< ref "user-guide/ci" >}}) to see the proper way to handle secrets in your CI environment to store your API key securely.
+{{< /alert >}}
+
+## Using your API key
+
+LocalStack expects your API key to be present in the environment variable `LOCALSTACK_API_KEY`. Before starting LocalStack, define the environment variable:
+
+{{< command >}}
+$ export LOCALSTACK_API_KEY=<your-api-key>
+{{< / command >}}
+
+### Starting LocalStack via CLI
+
+To start LocalStack using the LocalStack CLI, you don't have to perform any further steps (after exporting the environment variable).
+
+{{< command >}}
+$ localstack start
+{{< / command >}}
+
+LocalStack will detect the API key and properly pass it to the LocalStack container.
+
+### Starting LocalStack via Docker
+
+To start LocalStack using Docker, you have to specify the API key using the `-e` flag for environment variables:
+
+{{< command "hl_lines=5" >}}
+$ docker run \
+  --rm -it \
+  -p 4566:4566 \
+  -p 4510-4559:4510-4559 \
+  -e LOCALSTACK_API_KEY=${LOCALSTACK_API_KEY:- } \
+  localstack/localstack
+{{< / command >}}
+
+For more information about starting LocalStack, take a look at our general [Getting Started]({{< ref "getting-started" >}}) guide.
+
+### Starting LocalStack via Docker-Compose
+
+To start LocalStack using `docker-compose`, you have to include the `LOCALSTACK_API_KEY` environment variable in your `docker-compose.yml` file:
+
+```yaml
+environment:
+  - LOCALSTACK_API_KEY=${LOCALSTACK_API_KEY- }
+```
+
+It sets the API key we defined before (by using the `export` command) into your LocalStack container, such that the key activation can take place.
+
+## Licensing-related configuration
+
+If you want to make sure that LocalStack is only started if you can activate LocalStack Pro or Enterprise, or if you wish to suppress licensing-related error messages, take a look at our [configuration guide]({{< ref "configuration.md#localstack-pro">}}) regarding LocalStack Pro.
+
+## Checking license activation
+
+The easiest way to check if LocalStack is activated is to check the health endpoint of LocalStack for a list of the running services:
+
+{{< command >}}
+$ curl localhost:4566/health | jq
+{{< / command >}}
+
+If a Pro-only [service]({{< ref "aws" >}}) -- like [XRay]({{< ref "XRay-Tracing" >}}) -- is running, LocalStack has started successfully. Otherwise, check our collected most [common activation issues](#common-activation-issues).
```

Comment:
nit: We could also add a note here that users can check if the following line is present in the LocalStack container logs:
```
[...] Successfully activated API key
```